db.go

```diff
@@ -147,6 +147,27 @@ func (self *DB) Query(sql string, vars interface{}) (interface{}, error) {
 	return self.send("query", sql, vars)
 }
 
+// SchemalessSelect returns a list of rows from the database that have been added
+// to a table without a defined schema
+func (self *DB) SchemalessSelect(what string) ([]map[string]any, error) {
```

Comment:
Per Go conversions this should just be `db`. `self` and `this` is not used in Go.